docs/monitoring/infrastructure.md

```diff
@@ -0,0 +1,49 @@
+title: Sematext Core Infrastructure Monitoring
+description: Sematext Infrastructure Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
+
+
+Points to cover:
+
+## Overview
+
+Sematext Monitoring provides complete insight into your infrastructure - applications, servers, containers, processes, inventory, events...
+
+The centerpiece is newly introduced Infra App which acts as a container for all infrastructural data about your architecture. As with other App types, you can create as many Infra Apps under your account as you wish. Each Infra App would typically be used to gather data from one environment (for example, one Infra App for production environment, one for test environment and one for development environment).
+
+SematextCloud UI will automatically create one Infra App named "Default Infra App" the first time you might need it. You can also create Infra App on your own under Integrations->Overview.
+
+## How it works
+
+New Golang-based [Sematext agent](../agents/sematext-agent) (which is part of [spm-client](../agents/spm-client) installer) will be installed on your machine. It is sending all infrastructure data and metrics to token of one of your Infra Apps. This token is configured as part of installation instructions you can find in SematextCloud UI. It can be installed either on its own (in which case you would have only Infra monitoring enabled on particular machine) or while installing one of Sematext App Agents (for example Kafka or Elasticsearch agent) - SematextCloud UI always shows the instructions which include Infra App token.
```

Comment:
**A** * new Golang-based [Sematext agent](../agents/sematext-agent), which is part of the [spm-client](../agents/spm-client) installer, will be installed on your machine. **It's** * sending all infrastructure data and metrics **to a token that links** to one of your Infra Apps. This token is configured as part of **the** installation instructions you can find in Sematext Cloud. It can be installed either on its own, in which case you would **only have** Infra monitoring enabled on **a** particular machine, or while installing one of Sematext's App Agents (for example Kafka or Elasticsearch agent). Sematext Cloud always shows the instructions which include Infra App token.

Reply:
New => nothing.  It's new now, but it won't be new tomorrow. 

which is part of the spm-client installer => included in the spm-client package
It is sending all infrastructure data and metrics to token of one of your Infra Apps. This token is configured as part of installation instructions you can find in SematextCloud UI => This agent sends all infrastructure data to an Infra App whose token is configured during agent installation.

I think the rest of those instructions are too detailed, too specific.